prepare_inventory.py

```diff
@@ -0,0 +1,47 @@
+#!/bin/env python3
+
+# This is simple script which converts EDPM inventory from the OpenShift's secret
+# to the very simple inventory file which can be used by the RHOSO break/fix training
+# scripts while preparing training scenarios.
+# Usage of the script:
+# python3 prepare_inventory.py <path_to_the_edpm_inventory.yaml> <path_to_the_output_file.yaml>
+#
+# Edpm inventory file can be get from the OpenShift with command like:
+#
+# oc get secret <secret_name> -o json | jq '.data | map_values(@base64d)' | jq -r '.inventory'
+
+
+import yaml
+import sys
+
+
+if __name__ == '__main__':
+    src_file = sys.argv[1]
+    dst_file = sys.argv[2]
+    if not src_file:
```

Comment:
Is this possible? If the argument is omitted then the L19 is gonna throw an IndexError

Reply:
Actually it can only prevent in case when someone would pass empty string as argument. I added additional check just to be sure that there will be no such IndexError raised directly.